modules/about-sosreport.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="about-about-sosreport_{context}"]
```

Comment:
Link to or include some referance to https://github.com/openshift/openshift-docs/pull/23316/files#diff-10c10b678067946e6e4a477c54c42314

Reply:
In the 'Gathering data about your cluster' assembly, the 'Generating a sosreport archive for an OpenShift Container Platform cluster node' module directly follows the 'About sosreport' module. Currently it is not possibly to include `xref` references to other modules, within a module.